### PR TITLE
JSON builder

### PR DESCRIPTION
**Pull request by kimpenguin**
Added the JSON object builder with a link to it in the Endpoints page. Minimal styling applied. And no validation for the fields. Let me know if this is useful for our users.

### CODE REVIEW

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/request-builder.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    <article class="color-light-background py-2 px-4 screen-desktop:px-6">
+        <h1 id="request-builder">Request Builder</h1>
+<p>Use the following form to generate a data object you can pass to the Benefits Eligibility API.</p>
+
+  <form class="screener-form">
+    <div class="household-data">
+  <h2>Household</h2>
+  <p>Enter values into the fields below for the Household Level data for the Household Composition.</p>
+
+  <label>zip</label>
```
Comment (abourne1) [inline dist/68d422386f1c9b95dab97295f2644aa1687647a4/request-builder.html]: I think including a zip code field will cause the API to reject the request?

File: src/views/partials/household.twig [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<div class="household-data">
+  <h2>Household</h2>
+  <p>Enter values into the fields below for the Household Level data for the Household Composition.</p>
+
+  <label>zip</label>
```
Comment (abourne1) [inline src/views/partials/household.twig]: I think zip has to be removed from the request or else the API will return a 4xx

**Review by abourne1 — COMMENTED** [review]
visual nit: could you add a bit of padding to the buttons near the button of the json body created by the form?

screen shot 2019-02-11 at 5 30 01 pm

**Comment by abourne1** [comment]
Do you think it's worth surfacing an error if a user doesn't fill in a required field?

I think this is a great idea though! We have a lot of fields and if I was using this API for the first time I think I'd want something like this to show me how to generate a valid request.

File: src/views/partials/household.twig (hunk 2 above)
Comment (kimpenguin) [reply src/views/partials/household.twig]: you are absolutely right.

**Comment by kimpenguin** [comment]
@abourne1 Yeah, the styling needs a bit of work. *wink wink nudge nudge* @devowhippit 😆  
Just laying the foundation for the JSON build.

I'll get rid of zip. I've yet to actually test it as the body in the request to the API. I'll work in some validation and options too while I'm at it.

**Review by devowhippit — APPROVED** [review]

**Pull request merged**